cypress/support/e2e.js

```
@@ -18,3 +18,6 @@ import './commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+Cypress.on('uncaught:exception', (_err, runnable) => {
+  return false;
+});
```

This global handler currently swallows all uncaught exceptions by returning false. That can hide actual application errors and lead to false-positive test results. Consider filtering only known benign errors (e.g., check _err.message) or add a clear comment explaining why the blanket suppression is required. You can also gate this behavior behind an environment flag so CI runs can be stricter.